From: Hannes Reinecke <hare@suse.de>
Subject: Fixup kvm for s390
Patch-Mainline: not yet

These functions have been moved to the generic header, defining
it here breaks the build.

Signed-off-by: Hannes Reinecke <hare@suse.de>
Index: linux-3.7-rc2-master/arch/s390/include/asm/kvm_para.h
===================================================================
--- linux-3.7-rc2-master.orig/arch/s390/include/asm/kvm_para.h
+++ linux-3.7-rc2-master/arch/s390/include/asm/kvm_para.h
@@ -143,15 +143,4 @@ static inline int kvm_para_available(voi
 	return 1;
 }
 
-/* No feature bits are currently assigned for kvm on s390 */
-static inline unsigned int kvm_arch_para_features(void)
-{
-	return 0;
-}
-
-static inline bool kvm_check_and_clear_guest_paused(void)
-{
-	return false;
-}
-
 #endif /* __S390_KVM_PARA_H */
